invenio_jsonschemas/views.py

```diff
@@ -26,7 +26,15 @@
 
 from __future__ import absolute_import, print_function
 
-from flask import Blueprint, abort, send_from_directory
+from flask import (
```

Comment:
Please use `isort` configuration from `.editorconfig` which should be default. Thanks